src/Tools/dotnet-watch/BrowserRefresh/src/Microsoft.AspNetCore.Watch.BrowserRefresh.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- This feature is supported in projects targeting 3.1 or later.-->
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <Nullable>enable</Nullable>
+    <IsShipping>false</IsShipping>
+    <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
```

Comment:
Since this project is used from other implementation projects, it should be a project reference provider. Remove this line and run eng/scripts/GenerateProjectList.ps1

Reply:
Can you elaborate on what the benefit is of using one over the other? I thought it primarily helped with patch scenarios, but that wouldn't apply in this case. These middleware will always ship with the tool.

Reply:
We always ship everything except targeting packs when servicing. This is about consistency, special cases that are handled in eng/targets/ResolveReferences.targets only for project reference providers, centralizing information about these projects (allowing easier moves), and (this is a nit) shortening our project files.

Reply:
Done